docs/mobile-apps/automated-testing/about-appium.md

```diff
@@ -0,0 +1,439 @@
+---
+id: about-appium
+title: Using Appium for Automated Mobile App Testing
+sidebar_label: About Appium
```

Comment:
@walkerlj0 for now, let's copy all of Wim's suggestions in a JIRA ticket and assign it to Kim (the content owner). Also I don't think 'About Appium' is a good navbar label for this doc because currently it's all over the place and not a high-level doc about Appium (which should be the goal).

We should also consider breakiing out these topics into separate child docs. This is how I'm thinking it should be off the top of my head:

1. Install and Configure Appium
2. Create an Appium Test
3. Working with Appium Locators
4. Performing Actions with Appium
5. Handling Application Response and Test Results